gromacs/fileformats/blocks.py

```diff
@@ -0,0 +1,461 @@
+
+
+#
+# JD: adopted from 'pytopol' from https://github.com/resal81/PyTopol
+#
+
+class System(object):
+    def __init__(self):
+        self.molecules = tuple([])
+
+        self.atomtypes        = []
+        self.bondtypes        = []
+        self.nonbond_params   = []
+        self.angletypes       = []
+        self.dihedraltypes    = []
+        self.impropertypes    = []
+        self.cmaptypes        = []
+        self.interactiontypes = []
+        self.pairtypes        = []
+        self.constrainttypes  = []
+        self.forcefield=  None
+
+        self.information = {} # like 'atomtypes': self.atomtypes
+
+
+
+class Molecule(object):
+
+    def __init__(self):
+        self.chains    = []
+        self.atoms     = []
+        self.residues  = []
+
+        self.bonds     = []
+        self.angles    = []
+        self.dihedrals = []
+        self.impropers = []
+
+        self.cmaps     = []
+        self.pairs     = []
+        self.exclusion_numb = None  # 0, 1, 2, ..
+        self.virtual_sites3 = []
+        self.exclusions = []
+        self.settles    = []
+        self.constraints= []
+
```

Comment:
@MichalKononenko , don't worry!!! Code review is frank and it is always very helpful to have multiple pairs of eyes. Thanks for contributing to the project!